proto/babylon/zoneconcierge/v1/zoneconcierge.proto

```diff
@@ -131,9 +131,9 @@ message ProofFinalizedChainInfo {
     The following fields include proofs that attest the chain info is
     BTC-finalised
   */
-  // proof_cz_header_in_epoch is the proof that the CZ header is timestamped
+  // proof_bsn_header_in_epoch is the proof that the BSN's header is timestamped
   // within a certain epoch
-  tendermint.crypto.ProofOps proof_cz_header_in_epoch = 1;
+  tendermint.crypto.ProofOps proof_bsn_header_in_epoch = 1;
```

Comment:
same here, `proof_consumer_header_in_epoch` is better

Also this is going to be a breaking change :thinking:

Reply:
Yes, I know... the option here is to keep the old field name, just for compatibility reasons.

Reply:
sure, but maybe leave it as a TODO?

Reply:
Then I would say let's just do this change now, as in the future it'll be worse.